apps/full-site-editing/full-site-editing-plugin/premium-content/index.js

```diff
@@ -19,6 +23,10 @@ import * as loggedOutView from './blocks/logged-out-view';
 import * as buttons from './blocks/buttons';
 import * as loginButton from './blocks/login-button';
 
+const supportsDecoupledBlocks = !! (
+	__experimentalAlignmentHookSettingsProvider && __experimentalBlock
+);
```

Comment:
Thanks, I was starting to investigate experimental feature usage (https://github.com/Automattic/wp-calypso/issues/44251), I can tick these off.